client/src/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -14,6 +14,7 @@
   overflow: hidden;
   text-overflow: ellipsis;
   direction: rtl;
+  padding: 7px 0;
```

Comment:
I think a `padding-top: 7px;` is sufficient, unless you can see an instance where `padding-bottom: 7px` is also important?

shortcut css is nice, but it opens up the chance for leaking to or overriding other areas